src/cli/load-config.ts

```
@@ -24,6 +24,7 @@ import {
 } from '@shikijs/transformers'
 import { difference } from 'es-toolkit'
 import rehypeRaw from 'rehype-raw'
+import { cyan } from 'yoctocolors'
```

ESLint

[error] Unable to resolve path to module 'yoctocolors'.

(import-x/no-unresolved)